frontend/src/components/load-more/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component, PropTypes } from 'react'
+import { EVENTS_SEARCH_LIMIT } from 'api'
+import RaisedButton from 'material-ui/RaisedButton'
+
+export default class LoadMore extends Component {
+  static get propTypes () {
+    return {
+      loadMoreSearchResults: PropTypes.func.isRequired,
+      xhrStatus: PropTypes.shape({ lastEventsLoadSize: PropTypes.number }).isRequired
+    }
+  }
+
+  render () {
+    return (
+      this.props.xhrStatus.lastEventsLoadSize === EVENTS_SEARCH_LIMIT &&
```

Comment:
again, I prefer the guard clause instead of the booleanish if.

@tulios , thoughts?